gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -75,6 +75,19 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    gpii.ontologyHandler.tests.floydWarshallAlgorithmWithReconstruction = function () {
```

Comment:
We should have several more tests than this, preferably encoded into a suitable compact declarative form

Should test at a minumum:
The two different graphs with 3 vertices (2 edges and 3 edges)
At least 3 graphs with 4 vertices (square, tetrahedron and the one with 2 disconnected edges)